usaspending_api/download/filestreaming/download_generation.py

```diff
@@ -51,7 +51,7 @@ def generate_download(download_job):
     piid = json_request.get("piid", None)
     award_id = json_request.get("award_id")
     assistance_id = json_request.get("assistance_id")
-    extension = json_request.get("file_format")
```

Comment:
👍

Reply:
Aren't all of these data files the same "file format" simply with different delimiters and extensions?

Reply:
I would agree, but this stays consistent with the value that is passed in on Request. Needed to break apart the idea of the "file_format" and extension since "pstxt" is a "txt" file.

Reply:
An API is an abstraction layer for both directions :-)